AutomatedTesting/Gem/PythonTests/Atom/atom_utils/atom_constants.py

```diff
@@ -234,6 +236,7 @@ def hdri_skybox(property: str = 'name') -> str:
         """
         HDRi Skybox component properties.
         :param property: From the last element of the property tree path. Default 'name' for component name string.
+          - 'Cubemap Texture': Path to the asset file to use for the cubemap texture asset.
```

Comment:
Asset.id of the texture since this is not the path

Reply:
oh true that was a legit mistake ty